source/MaterialXContrib/MaterialXMaya/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ if (EXISTS "${MAYA_DEBUG_DIR}" OR EXISTS "${MAYA_RELEASE_DIR}")
         MaterialXGenShader
         MaterialXGenOgsXml
         MaterialXRender
+        $<$<BOOL:${MATERIALX_BUILD_CROSS}>:MaterialXCross>
```

Comment:
Only `MaterialXMaya` links against `MaterialXCross` and only if the feature is enabled.